Toluene/Burcat 1986 yaml files/Burcat_mixB.yaml

```diff
@@ -0,0 +1,177 @@
+---
+file-author:
+    name: Morgan Mayer
+    ORCID: 0000-0001-7137-5721
+file-version: 0
+chemked-version: 0.3.0
+reference:
+    doi: 19860015959
```

Comment:
I don't think this is a valid DOI?

Reply:
This Burcat paper did not have a doi since it was a technical memo but I did validate it with another valid doi from another paper, then I believe @kyleniemeyer used some get-around for using the files with pyteck. Should I take out the doi line for the Burcat files then?

Reply:
Yeah, I think that would be best until we decide what to do about papers like this without a permanent identifier.

Reply:
We had discussed using a URL for cases without a DOI; in this case, it would be https://ntrs.nasa.gov/search.jsp?R=19860015959 (so we know what that is moving forward)